src/components/footer/footer.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const Footer = () => {
+    return (
+        <footer className={'page-footer'} style={{
+            height: '50px',
```

Comment:
Having both css and styles inside a component is bad practice, please choose just one approach.